### PR TITLE
Analytics global object

### PR DESCRIPTION
**Pull request by harrisred**
* Update Skroutz Analytics tracking script to latest version
* Add ability to use a custom global object name

screen shot 2018-09-25 at 17 17 12

### CODE REVIEW

File: includes/class-wc-skroutz-analytics-integration.php [hunk 1]
```diff
@@ -132,6 +125,17 @@ public function init_form_fields() {
 				'placeholder' => 'custom_id',
 				'desc_tip'    => __( 'If custom id key is not found the Product ID/SKU will be used.', 'wc-skroutz-analytics' ),
 			),
+			'sa_custom_global_object_name_enabled' => array(
+				'title'    => __( 'Global Object Name', 'wc-skroutz-analytics' ),
+				'type'     => 'checkbox',
+				'label'    => __( 'Use custom global object name', 'wc-skroutz-analytics' ),
+				'desc_tip' => __( 'Change global object name if there is a conflict with the default value.', 'wc-skroutz-analytics' ),
```
Comment (clontos) [inline includes/class-wc-skroutz-analytics-integration.php]: [Minor] Change `if` with `when` in:
`Change global object name if...`

File: languages/wc-skroutz-analytics-el.po [hunk 2]
```diff
@@ -36,76 +36,99 @@ msgstr ""
...
 msgstr "Εάν δεν βρεθεί το custom id key θα χρησιμοποιηθεί το Product ID/SKU."
 
-#: includes/class-wc-skroutz-analytics-integration.php:150
+#: includes/class-wc-skroutz-analytics-integration.php:129
+msgid "Global Object Name"
+msgstr ""
+
+#: includes/class-wc-skroutz-analytics-integration.php:131
+msgid "Use custom global object name"
+msgstr "Χρήση custom global object name"
```
Comment (clontos) [inline languages/wc-skroutz-analytics-el.po]: `custom` -> `διαφορετικού`

File: languages/wc-skroutz-analytics-el.po [hunk 3]
```diff
@@ -36,76 +36,99 @@ msgstr ""
...
+msgstr ""
+"Αλλάξτε το global object name έαν χρησιμοποιείται ήδη στον κώδικα σας το "
+"προκαθορισμένο όνομα."
+
+#: includes/class-wc-skroutz-analytics-integration.php:136
+msgid ""
+"Specify a custom global object name that will be used in the analytics "
+"tracking code."
+msgstr ""
+"Eισάγετε ένα custom global object name που θα χρησιμοποιηθεί στον κώδικα του "
```
Comment (clontos) [inline languages/wc-skroutz-analytics-el.po]: `custom` -> `διαφορετικό`

File: languages/wc-skroutz-analytics-el.po [hunk 4]
```diff
@@ -36,76 +36,99 @@ msgstr ""
...
+"analytics tracking."
+
+#: includes/class-wc-skroutz-analytics-integration.php:156
 msgid "Shop Account ID is invalid"
 msgstr "Το Shop Account ID δεν είναι έγκυρο"
 
-#: includes/class-wc-skroutz-analytics-integration.php:164
-msgid "An error occurred. %s"
-msgstr "Προκλήθηκε ενα λάθος. %s"
```
Comment (clontos) [inline languages/wc-skroutz-analytics-el.po]: `Προκλήθηκε ` -> `Προέκυψε `

**Review by clontos — APPROVED** [review]
LGTM 👍

**Review by kosdiamantis — APPROVED** [review]
+1 LGTM

**Review by dkart — APPROVED** [review]

**Pull request merged**